include/seqan3/io/sequence_file/format_embl.hpp

```diff
@@ -274,31 +261,32 @@ class sequence_file_format_embl
         }
         else
         {
+            if (ranges::empty(id)) //[[unlikely]]
+                throw std::runtime_error{"The ID field may not be empty when writing embl files."};
+
             if (options.complete_header)
             {
                 ranges::copy(id, stream_it);
             }
             else
             {
-                if (ranges::empty(id)) //[[unlikely]]
-                    throw std::runtime_error{"The ID field may not be empty when writing embl files."};
-
                 stream_it = 'I';
                 stream_it = 'D';
                 stream_it = ' ';
                 ranges::copy(id, stream_it);
                 stream_it = ';';
                 stream_it = ' ';
                 ranges::copy(std::to_string(sequence_size), stream_it);
-                ranges::copy(std::string_view{" BP.\nXX\n"}, stream_it);
+                ranges::copy(std::string_view{" BP.\n"}, stream_it);
             }
 
         }
 
         // Sequence
         if constexpr (detail::decays_to_ignore_v<seq_type>) // sequence
         {
-            throw std::logic_error{"The SEQ and SEQ_QUAL fields may not both be set to ignore when writing embl files."};
+            throw std::logic_error{"The SEQ and SEQ_QUAL fields may not both be set to ignore when writing"
+            "embl files."};
```

Comment:
You need a space inside the string at the line break, otherwise you produce `writingembl`.